app/build.gradle

```diff
@@ -34,6 +34,8 @@ android {
 }
 
 dependencies {
+    implementation project(":shoplist")
```

Comment:
great addition!
added shoplist implementation after including the project in settings.gradle file